content/cloud-docs/api-docs/workspaces/workspace-tags.mdx

```diff
@@ -0,0 +1,235 @@
+---
+page_title: Workspace Tags - API Docs - Terraform Cloud and Terraform Enterprise
+---
+
+[200]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/200
+
+[201]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/201
+
+[202]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/202
+
+[204]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/204
+
+[400]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/400
+
+[401]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/401
+
+[403]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/403
+
+[404]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/404
+
+[409]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/409
+
+[412]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/412
+
+[422]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/422
+
+[429]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/429
+
+[500]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/500
+
+[504]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/504
+
+[JSON API document]: /cloud-docs/api-docs#json-api-documents
+
+[JSON API error object]: https://jsonapi.org/format/#error-objects
+
+[speculative plans]: /cloud-docs/run/#speculative-plans
+
+# Workspace Tags API
+
+## Get Tags
```

Comment:
Please check my permissions statement. But I think we should on each page give any needed permissions information for the endpoints on that page. 

```suggestion
# Workspace Tags API

Tags let you more easily [categorize, sort, and filter](/cloud-docs/workspaces#listing-and-filtering-workspaces) workspaces in Terraform Cloud. Managing tags requires permissions to read runs on the workspace. Refer to [Permissions](/cloud-docs/users-teams-organizations/permissions) for more details.

## Get Tags
```